### PR TITLE
Start moving some phase field documentation from old wiki to new site

### PR DESCRIPTION
**Pull request by jessecarterMOOSE**
Several helpful topics that were on the old phase-field wiki did not make it over to the new website. Some info may be outdated or converted incorrectly. I'm hoping to get more eyes on this before going too far.

Pages include:

- [x] Units
- [x] Anisotropy (<-- not much there?)
- [x] CALPHAD
- [x] Quantitative Two Component Polynomial Free Energies
- [x] Mechanics coupling
- [x] Elastic Driving Force For Grain Growth
- [x] Initial Conditions
- [x] Tutorial
- [x] Image Reader (<-- not much there?)

ping @dschwen @laagesen 

ref #14988

### CODE REVIEW

**Review by laagesen — CHANGES_REQUESTED** [review]
Thanks for doing this @jessecarterMOOSE !

**Comment by jessecarterMOOSE** [comment]
Thanks for the comments @laagesen . I'll fix those up.

By the way, is there a good reference for the `PolynomialFreeEnergy` and `PFParamsPolyFreeEnergy` materials? There quite a bit of derivation in docs so a reference would be good to add while I'm in there.

**Comment by jessecarterMOOSE** [comment]
OK, now I see there's a reference to Moelans et al. 2008, but after a quick search there are at least 3 articles in 2008 with Moelans as the first author lol

**Comment by laagesen** [comment]
I think that the Moelans 2008 paper is the one that develops the grain growth model used in MOOSE:

N. Moelans, B. Blanpain, and P. Wollants, Phys. Rev. B 78, 024113 (2008).

**Comment by jessecarterMOOSE** [comment]
> N. Moelans, B. Blanpain, and P. Wollants, Phys. Rev. B 78, 024113 (2008).
Thanks. I'll link to that paper.

What about the polynomial approximations to the logarithmic free energy function? Is there a reference for that? Is it just based on a Taylor series?

File: modules/phase_field/doc/content/modules/phase_field/Phase_Field_Model_Units.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+	F = \int_V \big[ f_{loc}(c_1, \ldots,c_N, \eta_1, \ldots, \eta_M) + f_{gr}(c_1, \ldots,c_N, \eta_1,  \ldots, \eta_M) + E_{d} \big] \, dV,
+\end{equation}
+
+where $f_{loc}$ defines the local free energy density with the units $e/l^3$. The gradient energy density
+
+\begin{equation}
+	f_{gr} = \sum_i^N \frac{\kappa_i}{2} |\nabla c_i|^2 + \sum^M_j \frac{\kappa_j}{2} |\nabla \eta_j|^2
+\end{equation}
+
+where $\kappa_i$ and $\kappa_j$ are gradient energy coefficients with the units $e/l$.  Finally, $E_d$ describes any additional sources of energy in the system, such as deformation or electrostatic energy, with units of $e/l^3$.
```
[the remark this answers is not in the document]
Comment (jessecarterMOOSE) [reply modules/phase_field/doc/content/modules/phase_field/Phase_Field_Model_Units.md:28]: Didn't realize that. What would this expression be if it were concentration?

[the remark this answers is not in the document]
Comment (laagesen) [reply modules/phase_field/doc/content/modules/phase_field/Phase_Field_Model_Units.md:28]: Well, I guess the equations would be the same actually if you used concentration, you would just have different units for kappa_c. Most people use mole fractions in phase-field modeling so I don't think you should necessarily give units for other cases, just mention that c is a mole fraction

**Comment by laagesen** [comment]
I think @dschwen came up with the polynomial approximation to the logarithmic function. It is based on the 3rd order Taylor series approximation. He can probably comment on whether anything should be cited for that

**Comment by dschwen** [comment]
IMO all images need to go into the `large_media` submodule.

**Comment by dschwen** [comment]
The polynomial approximation to the log goes back to Mike Tonks.

**Comment by jessecarterMOOSE** [comment]
> IMO all images need to go into the `large_media` submodule.

Does this include existing image files?

**Comment by dschwen** [comment]
> Does this include existing image files?

That ship has sailed. Once added to the main repo the images will remain in the history, whether we move them into the submodule or not. And thus they will contribute to the size of a full repo clone.

File: modules/phase_field/doc/content/modules/phase_field/Initial_Conditions.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+### Inputs
+
+- `x1`: The x coordinate of the lower left-hand corner of the box
+- `y1`: The y coordinate of the lower left-hand corner of the box
+- `z1`: The z coordinate of the lower left-hand corner of the box
+- `x2`: The x coordinate of the upper right-hand corner of the box
+- `y2`: The y coordinate of the upper right-hand corner of the box
+- `z2`: The z coordinate of the upper right-hand corner of the box
+- `inside`: The value of the variable inside the box
+- `outside`: The value of the variable outside the box
```
Comment (jessecarterMOOSE) [inline modules/phase_field/doc/content/modules/phase_field/Initial_Conditions.md:21]: Should the argument list (this object and others) be removed and replaced with a link to the syntax for the individual objects?

File: modules/phase_field/doc/content/modules/phase_field/Anisotropy.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+## `MatAnisoDiffusion` diffusion kernel
+
+## `SplitCHWResAniso` split Chan-Hilliard kernel
+
+## `ConstantAnisotropicMobility` mobility material
+
+## `CompositeMobilityTensor`
+
+## `GBAnisotropy`
```
Comment (jessecarterMOOSE) [inline modules/phase_field/doc/content/modules/phase_field/Anisotropy.md:17]: Suggestions for this page?

File: modules/phase_field/doc/content/modules/phase_field/CALPHAD.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# CALPHAD
```
Comment (jessecarterMOOSE) [inline modules/phase_field/doc/content/modules/phase_field/CALPHAD.md:1]: Is this page still valid?

**Review by jessecarterMOOSE — COMMENTED** [review]
> That ship has sailed. Once added to the main repo the images will remain in the history, whether we move them into the submodule or not. And thus they will contribute to the size of a full repo clone.

Those images are already my branch history so is the idea to (soft) reset back to devel, re-add just the *.md files, and then make a PR to large_media with the images?

File: modules/phase_field/doc/content/modules/phase_field/CALPHAD.md (hunk 4 above)
Comment (laagesen) [reply modules/phase_field/doc/content/modules/phase_field/CALPHAD.md:1]: Yes it should be. @dschwen recently submitted a manuscript about the use of this capability in conjunction with a new sublattice KKS phase-field model

**Review by laagesen — DISMISSED** [review]

**Comment by jessecarterMOOSE** [comment]
OK this PR is now markdown only. Images were submitted in idaholab/large_media#10.

**Comment by jessecarterMOOSE** [comment]
Updated the `large_media` submodule hash now that that PR is merged. This should be ready to go @dschwen @laagesen .

Some of the pages are still bare or maybe outdated, but that can be addressed as a follow-on to this PR.

**Comment by loganharbour** [comment]
@jessecarterMOOSE can you resolve the large_media conflict?

**Comment by jessecarterMOOSE** [comment]
ok @loganharbour fixed the conflict by moving the large_media submodule hash to the latest commit

**Comment by loganharbour** [comment]
Whenever a PR goes to civet to be tested, it is tested against devel. That is, we take devel and then throw your commits on top. What happened here is that BISON was failing with current devel at the time that your job started. Luckily, it was patched recently and should be good now.

**Comment by jessecarterMOOSE** [comment]
Looks good @loganharbour :+1:

**Comment by loganharbour** [comment]
@dschwen or @laagesen wanna give a final ✅ ?

**Comment by laagesen** [comment]
Looks good to me- thanks again @jessecarterMOOSE !

**Comment by jessecarterMOOSE** [comment]
And thanks to you guys for putting together nice documentation in the first place. Definitely something that should stay available.

File: modules/phase_field/doc/content/modules/phase_field/index.md [hunk 5]
```diff
@@ -49,21 +50,20 @@ MOOSE provides capabilities that enable the easy development of multiphase field
...
-- Initial Conditions: Basic phase field initial conditions
+- [Initial Conditions](Initial_Conditions.md): Basic phase field initial conditions
 - [Polycrystal Initial Conditions](ICs/PolycrystalICs.md)
 - Image Reader: Reconstructing initial conditions from images (SEM, optical, etc.)
 - [EBSD Reader](ICs/EBSD.md): Reconstructing initial conditions from EBSD and EDS data
 
 # Tutorials
 
-- Fe-Cr Phase Decomposition: Illustrates using parsed function kernels to create a two phase decomposition simulation
+- [Fe-Cr Phase Decomposition](Tutorial.md): Illustrates using parsed function kernels to create a two phase decomposition simulation
```
Comment (loganharbour) [inline modules/phase_field/doc/content/modules/phase_field/index.md:64]:
```suggestion
- [Fe-Cr Phase Decomposition](phase_field/Tutorial.md): Illustrates using parsed function kernels to create a two phase decomposition simulation
```

This is a pretty common filename

File: modules/phase_field/doc/content/modules/phase_field/index.md [hunk 6]
```diff
@@ -49,21 +50,20 @@ MOOSE provides capabilities that enable the easy development of multiphase field
 - [Grain Growth Model](Grain_Growth_Model.md): Background on the phase field model implemented in MOOSE
 - [Grain Tracker Algorithm](/GrainTracker.md)
 - [Grain Boundary Anisotropy](Grain_Boundary_Anisotropy.md)
-- Elastic Driving Force For Grain Growth
+- [Elastic Driving Force For Grain Growth](Elastic_Driving_Force_Grain_Growth.md)
 
 ## Initial Conditions
 
-- Initial Conditions: Basic phase field initial conditions
+- [Initial Conditions](Initial_Conditions.md): Basic phase field initial conditions
```
Comment (loganharbour) [inline modules/phase_field/doc/content/modules/phase_field/index.md:57]:
```suggestion
- [phase_field/Initial_Conditions.md]: Basic phase field initial conditions
```

File: modules/phase_field/doc/content/modules/phase_field/index.md [hunk 7]
```diff
@@ -49,21 +50,20 @@ MOOSE provides capabilities that enable the easy development of multiphase field
 - [Grain Growth Model](Grain_Growth_Model.md): Background on the phase field model implemented in MOOSE
 - [Grain Tracker Algorithm](/GrainTracker.md)
 - [Grain Boundary Anisotropy](Grain_Boundary_Anisotropy.md)
-- Elastic Driving Force For Grain Growth
+- [Elastic Driving Force For Grain Growth](Elastic_Driving_Force_Grain_Growth.md)
```
Comment (loganharbour) [inline modules/phase_field/doc/content/modules/phase_field/index.md:53]:
```suggestion
- [Elastic_Driving_Force_Grain_Growth.md]
```

File: modules/phase_field/doc/content/modules/phase_field/index.md [hunk 8]
```diff
@@ -35,7 +36,7 @@ MOOSE provides capabilities that enable the easy development of multiphase field
 
 ## Multiphysics Coupling
 
-- Mechanics Coupling - Coupling phase field equations with mechanics
+- [Mechanics Coupling](phase_field/Mechanics_Coupling.md) - Coupling phase field equations with mechanics
```
Comment (loganharbour) [inline modules/phase_field/doc/content/modules/phase_field/index.md:39]:
```suggestion
- [phase_field/Mechanics_Coupling.md] - Coupling phase field equations with mechanics
```

File: modules/phase_field/doc/content/modules/phase_field/index.md [hunk 9]
```diff
@@ -16,10 +16,11 @@ this module is found below:
 - [Expression Builder](FunctionMaterials/ExpressionBuilder.md): Using automatic differentiation of free energy material objects
 - [Solving Phase Field Models](phase_field/Solving.md): Basic info about solving phase field models
 - [Function Material Kernels](phase_field/FunctionMaterialKernels.md): Working with Function Materials that carry around their own derivatives
-- Phase Field Model Units: Discussion of units in phase field models
-- Anisotropy: Support of anisotropic mobilities and interfacial energies
-- CALPHAD: Using thermodynamic databases to parameterize phase field models
-- Quantitative Two Component Polynomial Free Energies: Simple two component models using polynomial free energies
+- [Function Materials](phase_field/FunctionMaterials.md): Creating material properties based on the value of an arbitrary function expression
```
Comment (loganharbour) [inline modules/phase_field/doc/content/modules/phase_field/index.md:19]:
```suggestion
- [phase_field/FunctionMaterials.md]: Creating material properties based on the value of an arbitrary function expression
```

File: modules/phase_field/doc/content/modules/phase_field/index.md [hunk 10]
```diff
@@ -16,10 +16,11 @@ this module is found below:
...
+- [Phase Field Model Units](phase_field/Phase_Field_Model_Units.md): Discussion of units in phase field models
```
Comment (loganharbour) [inline modules/phase_field/doc/content/modules/phase_field/index.md:20]:
```suggestion
- [phase_field/Phase_Field_Model_Units.md]: Discussion of units in phase field models
```

File: modules/phase_field/doc/content/modules/phase_field/index.md [hunk 11]
```diff
@@ -16,10 +16,11 @@ this module is found below:
...
+- [Anisotropy](phase_field/Anisotropy.md): Support of anisotropic mobilities and interfacial energies
```
Comment (loganharbour) [inline modules/phase_field/doc/content/modules/phase_field/index.md:21]:
```suggestion
- [phase_field/Anisotropy.md]: Support of anisotropic mobilities and interfacial energies
```

File: modules/phase_field/doc/content/modules/phase_field/index.md [hunk 12]
```diff
@@ -16,10 +16,11 @@ this module is found below:
...
+- [CALPHAD](phase_field/CALPHAD.md): Using thermodynamic databases to parameterize phase field models
```
Comment (loganharbour) [inline modules/phase_field/doc/content/modules/phase_field/index.md:22]:
```suggestion
- [phase_field/CALPHAD.md]: Using thermodynamic databases to parameterize phase field models
```

File: modules/phase_field/doc/content/modules/phase_field/index.md [hunk 13]
```diff
@@ -16,10 +16,11 @@ this module is found below:
...
+- [Quantitative Two Component Polynomial Free Energies](phase_field/Quantitative.md): Simple two component models using polynomial free energies
```
Comment (loganharbour) [inline modules/phase_field/doc/content/modules/phase_field/index.md:23]:
```suggestion
- [phase_field/Quantitative.md]: Simple two component models using polynomial free energies
```

File: modules/phase_field/doc/content/modules/phase_field/Anisotropy.md [hunk 14]
```diff
@@ -0,0 +1,17 @@
+# Anisotropy
```
Comment (loganharbour) [inline modules/phase_field/doc/content/modules/phase_field/Anisotropy.md:1]: As this isn't complete, can you put a filler in here?

**Review by loganharbour — CHANGES_REQUESTED** [review]
All minor changes.

Mostly taking advantage of the link-title-from-header feature in order to keep documentation consistent in the future.

File: modules/phase_field/doc/content/modules/phase_field/Anisotropy.md (hunk 14 above)
Comment (jessecarterMOOSE) [reply modules/phase_field/doc/content/modules/phase_field/Anisotropy.md:1]: Would it make sense to just list the syntax for these objects? What's that extension called?

Comment (jessecarterMOOSE) [reply modules/phase_field/doc/content/modules/phase_field/Anisotropy.md:1]: nvm, added some links and basic descriptions

**Comment by jessecarterMOOSE** [comment]
OK @loganharbour how's this? I used more autolinking and added more to the anisotropy page, plus fleshed out descriptions throughout.

**Review by loganharbour — APPROVED** [review]

**Comment by jessecarterMOOSE** [comment]
Thanks everyone!

**Comment by laagesen** [comment]
Thank you @jessecarterMOOSE!

**Pull request merged**